src/api/applicant/content-types/applicant/lifecycles.js

```diff
@@ -1,14 +1,54 @@
+const _ = require('lodash');
+const fetch = require('node-fetch');
 const { createModuleResolutionCache } = require("typescript");
+const { Webhook, MessageBuilder } = require('discord-webhook-node');
+// create new Webhook
+const hook = new Webhook(
+  `${process.env.OPEN_POSITIONS_DISCORD_WEBHOOK}`
+);
+
+hook.setUsername("New Applicant Notifier")
+.setAvatar('https://avatars.githubusercontent.com/u/53379976?s=200&v=4');
+
+const formatJSON = (jsonData) => {
+  return `\`\`\`JSON\n${JSON.stringify(jsonData, null, 1)}\n\`\`\``;
+};
+
+
 
 module.exports = {
 
-    async beforeCreate(event) {
+  async beforeCreate(event) {
+    // get the project associated with this applicant from the db
     const project = event.params.data.project;
     const applicantProject = await strapi.db.query('api::project.project').findOne({
-        where: {slug: project}
-      });
-    const projectID = applicantProject.id;
-    event.params.data.project = projectID
+      where: { id: project }
```

Comment:
I thought the frontend only passes the project name.

Reply:
I think there's might be a difference in the data being returned when a new applicant is created using the Strapi UI and the data returned when a new applicant is created using the POST request with the structure of the "request body" gotten from the frontend

Reply:
Bump this comment. I think we still need to query the project by name(slug).